### PR TITLE
OPSEXP-1944 Switch to new acs license access & secret keys

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-1944

### CODE REVIEW

**Review by gionn — COMMENTED** [review]
LGTM, adding Domenico and Damian to ACK before proceeding

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -389,8 +389,8 @@ jobs:
       - name: "Configure AWS credentials"
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.AWS_S3_ACSLICENSE_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_S3_ACSLICENSE_SECRET_ACCESS_KEY }}
```
Comment (dsibilio) [inline .github/workflows/ci.yml:393]: These credentials need to be added for DependaBot access as well, otherwise DependaBot PRs will fail.

**Review by dsibilio — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gionn) [reply .github/workflows/ci.yml:393]: Thanks Domenico 🙏🏻 

Screenshot 2023-02-28 at 09 46 39

Comment (dsibilio) [reply .github/workflows/ci.yml:393]: Thank **you!** 🙏

**Review by dsibilio — APPROVED** [review]

**Review by damianujma — APPROVED** [review]

**Pull request merged**